include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -1204,12 +1204,27 @@ def err_forany_type_variable_identifier_expected : Error<
 def err_forany_unexpected_token : Error<
   "unexpected token in _For_any specifier">;
 
+def err_itypeforany_comma_or_parenthesis_expected : Error<
```

Comment:
Clang development guidelines favor sharing diagnostic messages when possible, to minimize the number of different messages to manage.   Diagnostic message can contain formatting specifiers that allow the message to be customized for a specific circumstance.  We should try to follow those guidelines.